apcd-cms/src/apps/registrations/templates/submission_form/registration_form_scripts.html

```diff
@@ -267,7 +365,7 @@ <h6>
                 class="telephoneinput"
                 id="contact_phone_${contacts}{% if r %}_{{r.reg_id}}{% endif %}"
                 inputmode="tel"
-                pattern="^(\\+0?1\\s)?\\(?\\d{3}\\)?[\\s.-]\\d{3}[\\s.-]\\d{4}$"
+                pattern="^(\\+0?1\\s)?\\(?(?![{}'"<>\\/])\\d{3}\\)?(?![{}'"<>\\/])[\\s.-](?![{}'"<>\\/])\\d{3}(?![{}'"<>\\/])[\\s.-](?![{}'"<>\\/])\\d{4}$"
```

Comment:
Had to update the pattern for the telephone number input when adding more than one contact. It was the only field still allowing invalid characters on submission.

<img width="361" alt="Screenshot 2023-08-28 at 5 51 02 PM" src="https://github.com/TACC/Core-CMS-Custom/assets/96220951/e101c560-a37f-48f8-96eb-727619bbc036">